### PR TITLE
Show git revision in about dialog

### PR DESCRIPTION
**Pull request by rnestler**
See #17.

### CODE REVIEW

**Comment by rnestler** [comment]
Is this mergable as is or do I need to change something?

File: librepcb/controlpanel/controlpanel.cpp [hunk 1]
```diff
@@ -294,9 +294,11 @@ void ControlPanel::projectEditorClosed() noexcept
...
 void ControlPanel::on_actionAbout_triggered()
 {
-    QMessageBox::about(this, tr("About"), tr(
-        "LibrePCB is a free & open source schematic/layout-editor."
-        "<br><br>"
+    QMessageBox::about(this, tr("About LibrePCB"), tr(
+        "<h1>About LibrePCB</h1>"
+        "<p>LibrePCB is a free & open source schematic/layout-editor.</p>"
+        "<p>Version: " GIT_VERSION "</p>"
+        "<p>Please see <a href='https://librepcb.org/'>librepcb.org</a> for more information.</p>"
```
Comment (ubruhin) [inline librepcb/controlpanel/controlpanel.cpp]: Unfortunately librepcb.org does not support https, so the link does not work...

Comment (rnestler) [inline librepcb/controlpanel/controlpanel.cpp]: Ohh. Then get https working :wink:

Comment (ubruhin) [inline librepcb/controlpanel/controlpanel.cpp]: Will you pay the SSL certificate? ;)

Comment (dbrgn) [inline librepcb/controlpanel/controlpanel.cpp]: - Free certificates: https://startssl.com/
- 4$ certificates: https://gogetssl.com/domain-validation/domain-ssl/
- ...and many more.

Github pages doesn't support certificates on custom domains though. I could host the content & certificate for you, but then we wouldn't have auto-updating on push anymore.

Comment (ubruhin) [inline librepcb/controlpanel/controlpanel.cpp]: I also have a webspace, that's not the problem. But I don't have the time and nerves to set up the certificate, and even less to set up an auto-updating mechanism...

But as the website contains only static content and thus no sensitive data is transmitted, I would say SSL is not that important (at least until now). There are many other issues which are much more important ;)

**Pull request merged**